### DISCUSSION TITLE
Teleport Connect and ServiceNow integration

### DISCUSSION THREAD

**Discussion by jerh616** [gravitational/teleport#34887] [Q&A]
Hi there,

I've Teleport Connect working successfully with bastion in an environment with no integration but I cannot see how to pass a request-reason using the app so that i can access environments where the Bastion is configured with ServiceNow integration. I get a list of available clusters but permission denied when trying to get onto them (because, obviously enough, I have not successfully passed in the required parameters). All works fine using tsh on the cli. Both are v13.4.

Is this possible? There is nothing about this in the dox (https://goteleport.com/docs/ver/13.x/connect-your-client/teleport-connect/#troubleshooting) or the app config.

it would be great if this were possible as the app is really handy

Thanks folks

**Comment by zmb3** [comment]
Sounds like you're looking for the ability to submit access requests?

If so, that's accessible under the menu on the top right.

<img width="1392" alt="image" src="https://github.com/gravitational/teleport/assets/7527103/984dceae-eb6e-4106-86d3-580c3792df53">

<img width="1392" alt="image" src="https://github.com/gravitational/teleport/assets/7527103/76a91e1a-8f5c-4491-b6f6-3b87caf79723">
